### DISCUSSION TITLE
React hook form + react-input-mask not resetting input

### DISCUSSION THREAD

**Discussion by kuttymbekov** [react-hook-form/react-hook-form#3493] [Q&A] [answered]
I need to send email, if status is OK then reset field. but after submit, input phone mask is not resetting. and Strrange behaviour

https://codesandbox.io/s/dreamy-architecture-2xobo?file=/src/form.js

![Screenshot_1](https://user-images.githubusercontent.com/34336498/99879981-be750f00-2c3a-11eb-972f-4c3cc6f3f61a.png)

**Comment by SergeyWebDeveloper** [comment, marked as answer]
You have to give the default value to the controller which you have commented out
https://codesandbox.io/s/stoic-bohr-ozymp?file=/src/form.js

**Reply by bluebill1049** [reply]
Thank you for answering :)
